doc/guidelines/frontend-api-calls.md

```diff
@@ -86,3 +89,76 @@ const addFile = useMutation({
     },
   });
 ```
+
+### Scalable Approach
```

Comment:
I really like this approach. It also gives me a better understanding of the advantages of using useQuery.